doc/security/security-overview.rst

```diff
@@ -536,76 +536,190 @@ considered and documented.
-   entry invisible to anyone except as described below.
-
--  Security sensitive bugs are only accessible (view/modify) to members
-   of the Security Group; members of this Security Group are:
-
-   -  members of the Zephyr Security Subcommittee
-
-   -  others, as proposed and ratified by the Zephyr Security Subcommittee
-
-   -  the reporter
-
--  Members of this Security Group have the authority to add or remove other
-   users for individual issues.
-
--  Zephyr Security Subcommittee meetings have to review the embargoed bugs on
-   every meeting with more than three people in attendance. Said
-   review process shall decide if new issues needs to be embargoed
-   or not.
-
--  Security sensitive bugs shall be made public (by removing the
-   security sensitive indicator) after an embargo period of 60
-   days. The Zephyr Security Subcommittee is the only entity with authority
-   to extend the embargo period on a case by case basis; the JIRA
-   entry should be updated with the rationale for the embargo
-   extension so at some point said rationale will be made public.If
-   the Zephyr Security Subcommittee does not act upon a security sensitive
-   bug after its 60 days of embargo are over, it shall be
-   automatically made public by removing the security sensitive
-   setting.
-
--  Likewise, there shall be code repositories marked as security
-   sensitive, accessible only to the Security Group members where
-   the code to fix said issues is being worked on and reviewed. The
-   person/s contributing the fix shall also have access, but fix
-   contributors shall have only access to the tree for said fix, not
-   to other security sensitive trees.
-
--  A CVE space shall be allocated to assign Zephyr issues when the
-   Zephyr Security Subcommittee decides such is needed.
-
--  The severity of the issue with regard to security shall be entered by
-   the reporter.
-
--  All security relevant issues shall trigger an automated notification
-   on the Zephyr security mailing list
-   (vulnerabilities@zephyrproject.org).  Any member of the security
-   board can then triage the severity of the issue according to [CVSS]_.
-
--  Depending on the resulting severity score of the issue, the issue is
-   prioritized and assigned to the owner of the affected module.
-   Additionally, the system security architect and the security
-   architect of the module are notified and shall take the
-   responsibility to mitigate the issue and review the solution or
-   counter-measure. In any case, the security issue shall be
-   documented centrally, including the affected modules, software
-   releases, and applicable workarounds for immediate mitigation. A
-   list of known security issues per public release of the Zephyr
-   shall be published and maintained by the Zephyr Security Subcommittee after a
-   risk assessment.
+Vulnerabilities to the Zephyr project may be reported via email to the
```

Comment:
Suggest the first 2 paras go under the title "Security Vulnerability Reporting" and the "Security Issue Management" title is below those 2 paras.